### DISCUSSION TITLE
[matio] How could I introduce it to my project?

### DISCUSSION THREAD

**Discussion by ClaraSchumann** [microsoft/vcpkg#32538] [General]
I tried `find_package(matio)` `find_package(Matio)` `find_package(matio CONFIG)` and `find_package(Matio CONFIG)`. But none of them worked.

**Comment by Osyotr** [comment]
If upstream exports cmake config or pkg-config files, but vcpkg does not install, please create vcpkg issue.
If upstream does not do that, you should find things manually (for example, https://gist.github.com/Osyotr/709e71a91a3b277b01b684a0d638e758)
